app/kubemci/pkg/gcp/loadbalancer/loadbalancersyncer.go

```diff
@@ -244,6 +252,10 @@ func (l *LoadBalancerSyncer) DeleteLoadBalancer(ing *v1beta1.Ingress) error {
 		// Aggregate errors and return all at the end.
 		err = multierror.Append(err, hcErr)
 	}
+	if iErr := l.ins.DeleteIngress(ing, l.clients); iErr != nil {
```

Comment:
This reverses the order that we have today.
Today, we delete the ingress from clusters first and then delete GCP resources.

Not sure if this ordering change was intentional?

Reply:
I put it at the end so that it would be consistent with the comment a few lines above that states that resources are deleted in the reverse order in which they were created. I was aware that I was changing behavior, but I thought it made sense. Should it be deleted first?

Reply:
In the latest commit the old behavior has been restored, i.e. the ingress resources are deleted first.